README.md

```diff
@@ -1,131 +1,13 @@
-* [API:Allusers](https://www.mediawiki.org/wiki/API:Allusers)
-  * [get_allusers.py](python/get_allusers.py): get a list of all registered users, as ordered by username
-  * [get_allcategories.py](python/get_allcategories.py): get all categories that fit certain criteria relating to their titles
-* [API:Backlinks](https://www.mediawiki.org/wiki/API:Backlinks)
-  * [get_backlinks.py](python/get_backlinks.py): list pages which link to a certain page
-* [API:Random](https://www.mediawiki.org/wiki/API:Backlinks)
-  * [get_random.py](python/get_random.py): get a list of random pages
-* [API:Move](https://www.mediawiki.org/wiki/API:Move)
-  * [move.py](python/move.py): move a page
-* [API:Watch](https://www.mediawiki.org/wiki/API:Watch)
-  * [watch.py](python/watch.py): add a page to your watchlist 
-* [API:Alllinks](https://www.mediawiki.org/wiki/API:Alllinks)
-  * [get_alllinks.py](python/get_alllinks.py): list links to a namespace
-* [API:Exturlusage](https://www.mediawiki.org/wiki/API:Exturlusage)
-  * [get_exturlusage.py](python/get_exturlusage.py): enumerate pages that contain a given URL
-* [API:RecentChanges](https://www.mediawiki.org/wiki/API:RecentChanges)
-  * [get_recent_changes.py](python/get_recent_changes.py): get the three most recent changes with sizes and flags
-* [API:Querypage](https://www.mediawiki.org/wiki/API:Querypage)
-  * [get_querypage_list.py](python/get_querypage_list.py): List first 10 pages which are uncategorized
-* [API:SetPageLanguage](https://www.mediawiki.org/wiki/API:SetPageLanguage)
-  *  [set_page_language.py](python/set_page_language.py): change page language
-* [API:Embeddedin](https://www.mediawiki.org/wiki/API:Embeddedin)
-  * [get_embedded_pages.py](python/get_embedded_pages.py): get all page(s) that embed a page
-* [API:Rollback](https://www.mediawiki.org/wiki/API:Rollback)
-  * [rollback.py](python/rollback.py): rollback the last edits made to a given page
-* [API:Allfileusages](https://www.mediawiki.org/wiki/API:Allfileusages)
-  * [get_allfileusages.py](python/get_allfileusages.py): list of all file usages
-* [API:Protect](https://www.mediawiki.org/wiki/API:Protect)
-  * [protect.py](python/protect.py): Change the protection level of a given page
-* [API:Upload](https://www.mediawiki.org/wiki/API:Upload)
-  * [upload_file_directly.py](python/upload_file_directly.py): upload a file directly from the system
-  * [upload_file_from_url.py](python/upload_file_from_url.py): upload a file from a URL 
-  * [upload_file_in_chunks.py](python/upload_file_in_chunks.py): upload a file in chunks
-* [API:Pagepropnames](https://www.mediawiki.org/wiki/API:Pagepropnames)
-  * [get_pagepropnames.py](python/get_pagepropnames.py): List all page property names in use on the wiki
+# MediaWiki Action API Code Samples
+The MediaWiki action API is a web service that allows access to some wiki-features like authentication, page operations, and search. It can provide Meta information about the wiki and the logged-in user.
```

Comment:
MediaWiki Action API to be linked w/ https://www.mediawiki.org/wiki/API:Main_page.

Reply:
`M` in `Meta information..` to be in lowercase.